beacon-chain/core/helpers/committee_test.go

```diff
@@ -246,19 +246,19 @@ func TestAttestationParticipants_NoCommitteeCache(t *testing.T) {
 			attestationSlot: 3,
 			stateSlot:       5,
 			bitfield:        []byte{0x03},
-			wanted:          []uint64{127, 71},
+			wanted:          []uint64{71, 127},
 		},
 		{
 			attestationSlot: 2,
 			stateSlot:       10,
 			bitfield:        []byte{0x01},
-			wanted:          []uint64{85, 103},
+			wanted:          []uint64{85},
 		},
 		{
 			attestationSlot: 11,
 			stateSlot:       10,
 			bitfield:        []byte{0x03},
-			wanted:          []uint64{102, 68},
+			wanted:          []uint64{68, 102},
 		},
```

Comment:
Please add a case for bitfield: []byte{0x00} and wanted empty list.

Reply:
I added a test for that 
```
TestAttestationParticipants_EmptyBitfield
```

Reply:
Please add it here too

Reply:
OK it was merged anyway, i'll add it